msal/build.gradle

```diff
@@ -23,6 +23,22 @@ def bothTestType = "BothTest"
 
 android {
 
+    testOptions {
+        compileOptions {
+            // Flag to enable support for the new language APIs
+            coreLibraryDesugaringEnabled true
+            // Sets Java compatibility to Java 8
+            sourceCompatibility JavaVersion.VERSION_1_8
+            targetCompatibility JavaVersion.VERSION_1_8
+        }
+        dependencies {
+            coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:$rootProject.ext.coreLibraryDesugaringVersion"
+        }
+        defaultConfig {
+            multiDexEnabled true
+        }
```

Comment:
I see this got moved....I believe this is also only needed for test?

Reply:
I put it back.  It really doesn't matter for our libraries, since they don't produce APKs, but we might have a transitive dependency somewhere.